plugins/Installation/tests/System/APITest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Installation\tests\System;
+
+use Piwik\Config;
+use Piwik\Tests\Framework\Constraint\HttpResponseText;
+use Piwik\Tests\Framework\Fixture;
+use Piwik\Tests\Framework\TestCase\SystemTestCase;
+
+/**
+ * @group Installation
+ * @group APITest
+ * @group Plugins
+ */
+class APITest extends SystemTestCase
+{
+    /**
+     * @var Fixture
+     */
+    public static $fixture = null; // initialized below class definition
+
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass();
+
+        $testingEnvironment = new \Piwik\Tests\Framework\TestingEnvironmentVariables();
+        $testingEnvironment->configFileLocal = PIWIK_INCLUDE_PATH . '/plugins/Installation/tests/resources/config.ini.php';
+        $testingEnvironment->save();
+    }
```

Comment:
Should be possible to set configOverride this way. configFileLocal doesn't work because it's created before the DI container (since the INI config is needed before this point)... but there should ideally be only one way to set these properties...

Anyway, it's not important so I will merge this PR.